deploy/docker/README.md

```diff
@@ -27,6 +28,99 @@ $ docker run -d \
     --logtostderr=true \
     --containerized
 ```
+
+##### Docker for Mac/Windows
+```console
+# Fix mounting
+$ docker run --rm --volume=/:/rootfs:rw --pid=host --privileged gcr.io/k8s-minikube/localkube-amd64:LOCALKUBE_VERSION nsenter --mount=/proc/1/ns/mnt sh -c "if ! df | grep /var/lib/kubelet > /dev/null; then mkdir -p /var/lib/kubelet && mount --bind /var/lib/kubelet /var/lib/kubelet && mount --make-shared /var/lib/kubelet; fi"
+$ docker run -d \
+    --volume=/sys:/sys:ro \
+    --volume=/var/lib/docker:/var/lib/docker:rw \
+    --volume=/var/lib/kubelet:/var/lib/kubelet:shared \
+    --volume=/var/run:/var/run:rw \
+    --volume="${HOME}"/.minikube/certs:/var/lib/localkube/certs:rw \
+    --volume="${HOME}"/.minikube/etcd:/var/lib/localkube/etcd:rw \
+    --volume="${HOME}"/.minikube/manifests:/etc/kubernetes/manifests:ro \
+    --name=minikube \
+    --net=host \
+    --pid=host \
+    --privileged \
+    gcr.io/k8s-minikube/localkube-amd64:LOCALKUBE_VERSION \
```

Comment:
can we make this a ${LOCAKUBE_VERSION:-v1.5.3} or something similar so you can just copy and paste this command?